adr/ADR-41.md

```diff
@@ -77,24 +77,30 @@ Here we set the `msg_trace` configuration for the `A` account, this enables supp
 
 Note the `sampling`, here set to 100% which is the default, will further trigger only a % of traces that have the `sampled` value in the `traceparent` header.  This allow you to specifically sample only a subset of messages traversing NATS while your micro services will sample all.
 
-When this feature is enabled any message holding the `Nats-Trace-Dest` header as in the previous section will behave as if the `traceparent` header was not set at all.  In essense the Ad-Hoc mode has precedence.
+When this feature is enabled any message holding the `Nats-Trace-Dest` header as in the previous section will behave as if the `traceparent` header was not set at all.  In essence the Ad-Hoc mode has precedence.
 
 ### Cross Account Tracing
 
-By default a trace will end at an account boundary when crossing an Import or Export. This is a security measure to not allow an importing account to observe the internals of the exporting account.
-
-The exporting account can opt into this sharing using the configuration items below:
+By default a trace will end at an account boundary when crossing an Import or Export. This is a security measure to restrict visibility into foreign accounts and require opt-in to allow.
 
 ```
 accounts {
   B {
-   exports = [ { allow_trace: true, stream: "nats.add" } ]
+   exports = [ 
+    // on a service the direction of flow is into the exporting
+    // account, so the exporter need to allow tracing
+    { service: "nats.add", allow_trace: true } 
+   ]
+   
+   imports = [
+    // on a stream import the directin of flow is from exporter into
+    // the importer, so the importer need to allow tracing
+    {stream: {account: user, subject: ticker}, allow_trace: true}
```

Comment:
This should be account `A`, given the above code block? Should the `ticker` export be defined as well?

Reply:
It could be A, mauby be clearer. 

I am just showing that the thing that needs to be set it only on the importer of a `stream` import, the export doesnt change.

But on a `service` the export change not the import.

So only showing the bits that actually different from typical

Reply:
Ok, there is a chance it confuse the average reader, but we can allow a contribution to address if you don't want to fix it now.

Reply:
Well fully the idea is that these documents are not for the average reader, we absolultely need to treat these as input for general audience not output.

It's implementers and early adopters and deep technical reference.